scenedetect/video_splitter.py

```diff
@@ -227,15 +227,14 @@ def split_video_ffmpeg(input_video_paths, scene_list, output_file_template, vide
             call_list += [
                 '-y',
                 '-ss',
-                str(start_time.get_seconds()),
+                str(start_time.get_timecode()),
                 '-i',
-                input_video_paths[0]]
+                input_video_paths[0],
+                '-t',
```

Comment:
Does it matter if -t is specified before or after the codec?

Reply:
I don't think so, as it still needs to transcode through the video to this point, it can't do file-based seeking to get the end point like it does the start

Reply:
Oh, I misunderstood, hence talking of file-based seeking.

I don't think it matters if it's before or after arg_override, would you prefer it go after?

Reply:
No preference, was just wondering if it was intentional or not.  As-is is fine!  :)